meeteval/wer/__main__.py

```diff
@@ -477,6 +472,13 @@ def add_command(fn):
                          'equidistant_points: Place time points equally spaded int the segment-level intervals; '
                          'full_segment: Use the full segment for each word that belongs to that segment.'
                 )
+            elif name == 'hypothesis_allow_speaker_self_overlap':
+                command_parser.add_argument(
+                    '--hypothesis-allow-speaker-self-overlap', action='store_true',
+                    help='Allow speaker self-overlap in the hypothesis. '
+                         'This can change the order of words, so it is not recommended to use this option. '
+                         'It is not guaranteed that the returned WER is correct if this option is set!'
```

Comment:
Maybe give a motivation, when someone may set this option, e.g. lazy or preview, before fixing the system?
```
Allow speaker self-overlap in the hypothesis.
This can change the order of words, so it is not recommended to use this option.
You may set it, when you are too lazy to fix your system or you are eager and want to see a preview of your WER.
It is not guaranteed that the returned WER is correct if this option is set!
```

Reply:
Done